front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -110,6 +110,7 @@ const StdcmConfig = ({
       maxSpeed: validateMaxSpeed(maxSpeed, rollingStock?.max_speed),
     };
   }, [rollingStock, towedRollingStock, totalMass, totalLength, maxSpeed]);
+  const [validationMessage, setValidationMessage] = useState<string | null>(null);
```

Comment:
I think we can just use undefined
```suggestion
  const [validationMessage, setValidationMessage] = useState<string>();
```

Reply:
Good catch ! Done